Common/Basic components/TextBox.cs

```diff
@@ -7,14 +7,14 @@
 
 namespace Infrastructure
 {
-    public class TextBox : ComppenantBase
+    public class TextBox : ComponentBase
```

Comment:
You don't need to make this class in my opinion (unless there's a specific reason for doing so), each IWebElement can be sent keys to anyway